src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt

```diff
@@ -354,8 +354,8 @@ fun buildUpsertEntitiesAndLinkedData(): String {
  */
 val lockEntitiesInIdsTable =
         "SELECT 1 FROM ${IDS.name} " +
-                "WHERE ${ENTITY_SET_ID.name} = ? AND ${ID_VALUE.name} = ANY(?) AND ${PARTITION.name} = ? " +
-                "ORDER BY ${ID_VALUE.name} " +
+                "WHERE ${ENTITY_SET_ID.name} = ? AND ${ID_VALUE.name} = ? AND ${PARTITION.name} = ? " +
+//                "ORDER BY ${ID_VALUE.name} " +
```

Comment:
is this supposed to be commented out?

Reply:
oh i see

Reply:
Yes, but I should actually just remove the comment.

Reply:
How will the case where there are two batch writes at the same time writing to overlapping sets of entities work? (ie if write1 locks EKID1, write2 locks EKID2, write1 tries to lock EKID2, write2 tries to lock EKID1)

Reply:
ah i guess the sorted() should take care of this